src/braket/ir/jaqcd/program_v1.py

```diff
@@ -106,9 +117,9 @@ class Program(BraketSchemaBase):
     Attributes:
         braketSchemaHeader (BraketSchemaHeader): Schema header. Users do not need
             to set this value. Only default is allowed.
-        instructions (List[GateInstructions]): List of instructions.
-        basis_rotation_instructions (List[GateInstructions]): List of instructions for
-            rotation to desired measurement bases. Default is None.
+        instructions (List[Union[GateInstructions, NoiseInstructions]]): List of instructions.
+        basis_rotation_instructions (List[GateInstructions]): List of
```

Comment:
Nit: you can revert the changes in basis_rotation_instructions so it doesn't show up in the change history.

Reply:
The original changes came with other changes in the same file in the same commit. I don't find a good way to do it. Please let me know if there is a good way to do it, thanks!